### DISCUSSION TITLE
Ajout message validation lors de l'achat d'une potion dans le shop

### DISCUSSION THREAD

**Discussion by ghost** [Crownicles/Crownicles#859] [Nouvelles idées]
Je pense que je ne suis pas la seule à me faire avoir.
Si c'était possible d'ajouter un message pour signaler qu'il n'y a plus d'emplacement libre de dispo pour une nouvelle potion, siouplaît !

![image](https://user-images.githubusercontent.com/85878714/149357551-458962f9-79be-4346-b40f-ca53b7ac8315.png)

**Comment by Pietagorh** [comment]
Plutôt le rajouter en bas du message de validation actuel j'pense

**Reply by ghost** [reply]
ça me va du moment qu'il y a un warning sur le fait que la potion risque d'être gâchée !
